account_asset_management_xlsx/report/report_account_asset_xlsx.py

```diff
@@ -0,0 +1,458 @@
+        resources.update(
+            {
+                "row_pos": 5,
+                "workbook": workbook,
+            }
+        )
+        resources.update(self._define_formats(workbook))
+
+        sheet = workbook.add_worksheet()
+        resources.update({"sheet": sheet})
+
+        self._set_cells_size(resources)
+        self.generate_report_title(resources)
+
+        resources = self.generate_report_general(wizard_rec, resources)
+
+        profile_datas_lst = wizard_rec and wizard_rec.get_profile_datas() or []
+        resources.update(
+            {
+                "summary_column_info": {
+                    "value": [],
+                    "value_residual": [],
+                    "salvage_value": [],
+                    "amo_ant": [],
+                    "amo_de_lan": [],
+                    "cum_amo": [],
+                    "val_nette": [],
+                }
+            }
+        )
+        for profile_datas in profile_datas_lst:
+            resources = self.generate_report_profile(profile_datas, resources)
+
+        resources = self.generate_report_summary(resources)
+
+    def _set_cells_size(self, resources):
+        sheet = resources["sheet"]
+        sheet.set_default_row(20)
+        sheet.set_column("A:Z", None, resources["fm_default"])
+        sheet.set_column("A:A", 40)
+        sheet.set_column("B:L", 20)
+
+    def generate_report_title(self, resources):
+        logo_base64 = base64.b64decode(self.env.user.company_id.logo)
+        logo_image = image_process(logo_base64, size=(128, 128))
+        image_data = BytesIO(logo_image)
+        sheet = resources["sheet"]
+        sheet.insert_image(0, 0, "company_logo", {"image_data": image_data})
+        sheet.merge_range(
+            "B2:K5",
+            self.env._("Assets And Depreciation"),
+            resources["fm_report_title"],
+        )
+
+    def generate_report_general(self, wizard_rec, resources):
+        row_pos = resources["row_pos"]
+        col_pos = 0
+        created_infos = self._get_created_by_info(wizard_rec)
+
+        sheet = resources["sheet"]
+        sheet.write_rich_string(
+            row_pos,
+            col_pos,
+            resources["fm_bold"],
+            created_infos[0],
+            resources["fm_default"],
+            created_infos[1],
+        )
+
+        row_pos += 2
+        from_date_header = self.env._("Opening : ")
+        from_date_str = self.env["ir.qweb.field.date"].record_to_html(
+            wizard_rec, "from_date", {}
+        )
+        sheet.write_rich_string(
+            row_pos,
+            col_pos,
+            resources["fm_bold"],
+            from_date_header,
+            resources["fm_default"],
+            from_date_str,
+        )
+        row_pos += 1
+        to_date_header = self.env._("Closing : ")
+        to_date_str = self.env["ir.qweb.field.date"].record_to_html(
+            wizard_rec, "to_date", {}
+        )
+        sheet.write_rich_string(
```

Comment:
the closing date was not displayed on the report. There should be a row for it right above the line of asset profile.
<img width="589" height="149" alt="image" src="https://github.com/user-attachments/assets/6bcffab9-ed24-40d6-83a9-3be98c317cb7" />